dotcom-rendering/src/lib/assets.ts

```diff
@@ -9,18 +9,28 @@ interface AssetHash {
 	[key: string]: string;
 }
 
+export const BASE_URL_DEV = 'http://localhost:3030/';
+
 export type AssetOrigin =
 	| 'https://assets.guim.co.uk/'
 	| 'https://assets-code.guim.co.uk/'
+	| typeof BASE_URL_DEV
 	| '/';
 
 /**
  * Decides the url to use for fetching assets
  *
  * @param {'PROD' | 'CODE' | undefined} stage the environment code is executing in
- * @returns {string}
+ * @param {boolean} isDev whether the environment is development
+ * @returns {AssetOrigin}
  */
-export const decideAssetOrigin = (stage: string | undefined): AssetOrigin => {
+export const decideAssetOrigin = (
+	stage: string | undefined,
```

Comment:
If the stage is `undefined`, are we always executing in a development environment? I wonder if this can be simplified by returning `BASE_URL_DEV` in the `default` case in the `switch` condition below.